### DISCUSSION TITLE
RaspberryMatic aus Versehen in LAN-Gatway "verwandelt"

### DISCUSSION THREAD

**Discussion by UPaulick** [community/community#32426] [Other Feature Feedback, Questions, & Ideas]
Hallo Zusammen,
ich habe seit ein paar Jahren, zunächst mit CCU2 und dann mit einer RaspberryPI eine CCU3, also eine RapberryMatic am laufen. Bin kein Freak (im positiven Sinn, weil keine Zeit...) und wollte die alte CCU2 als LAN-Gatway einrichten, was nur "zur Hälfte funktionierte". Aus diesem Grunde habe nach Abhilfe gesucht und "in geistiger Umnachtung" mit dem Scipt:

string stdout;
string stderr;
system.Exec('touch /usr/local/HMLGW && reboot', &stdout, &stderr);

über "Programme und Verknüpfungen" meine "eigentliche Hauptstation" in ein LAN-Gateway verwandelt. Nun suche ich nach einer Möglichkeit, das wieder rückgängig zu machen. Habe - SSH ist dort aktiviert - nun mittels Putty versucht, die Datei "/usr/local/HMLGW" zu entfernen. Putty macht mir "die Tür zu meiner CCU auf", aber ich komme aber nicht rein. Das Passwort wird nicht angenommen. 
Was kann ich tun, kann mir hier jemand helfen???

Besten Dank in "die Runde", VG U. Paulick

PS. Sorry, da ich hier neu bin, habe ich keine Ahnung, welche Kategorie ich hier verwenden soll...

**Comment by airtower-luna** [comment]
Wahrscheinlich ist es am besten, wenn du direkt im RaspberryMatic Repository fragst: https://github.com/jens-maus/RaspberryMatic/discussions

Das hier ist die allgemeine GitHub Community, wo es hauptsächlich um Git und GitHub (und manchmal allgemeine Programmierfragen) geht. Kann natürlich sein daß jemand mit RaspberryMatic-Erfahrung mitliest, aber das ist halt Zufall.

**Reply by UPaulick** [reply]
Besten Dank, hoffe, es jetzt richtig gemacht zu haben...
